src/flamegraph/mod.rs

```diff
@@ -733,3 +758,22 @@ fn filled_rectangle<W: Write>(
     }
     svg.write_event(&cache_rect)
 }
+
+// Tries to find a sample count at the end of a line and returns its index.
+fn rfind_samples(line: &str) -> Option<usize> {
```

Comment:
Is there a way we can combine this with the corresponding function for extracting the sample used when we produce the flamegraph?